dotCMS/src/main/java/com/dotmarketing/portlets/templates/business/TemplateAPIImpl.java

```diff
@@ -44,16 +46,14 @@
 import java.util.stream.Collectors;
 
 import com.liferay.portal.model.User;
+import org.apache.velocity.tools.view.context.ViewContext;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.apache.velocity.tools.view.context.ViewContext'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)